src/planscape/forsys/views.py

```diff
@@ -277,20 +266,17 @@ def scenario_set(request: HttpRequest) -> HttpResponse:
 
     huc12_id = 43
 
-    project_area = Polygon( ((-120.14015536869722, 37.05413814388948),
-                             (-120.18409937110482, 36.9321584213366),
-                             (-119.93422142411087, 36.94003252840713),
-                             (-120.03710286062301, 36.99713288358574),
-                             (-120.14015536869722, 37.05413814388948)) )
-    project_area.srid = 4269
+    project_area = Polygon( ((-120.14015536869722, 39.05413814388948),
```

Comment:
could we add a comment describing what this code segment does?

Reply:
I've moved this into a new method, "fetch_input_params". Can that act as a comment, or should I elaborate?